install/firmware.csv

```diff
@@ -2,6 +2,30 @@ Make,Model,Serial,Version,Start Date,End Date,Notes
 "Allen Osborne Associates, Inc.",ROGUE SNR-8000,132,3.2,1995-05-31T00:00:00Z,1999-08-12T23:10:00Z,
 "Allen Osborne Associates, Inc.",ROGUE SNR-8000,132,3.2.32.8,1999-08-12T23:10:01Z,9999-01-01T00:00:00Z,
 "Allen Osborne Associates, Inc.",ROGUE SNR-8000,133,3.2.32.8,1995-01-01T00:00:00Z,9999-01-01T00:00:00Z,
+Campbell Scientific,CR6,3066,CR6.Std.04,2018-08-10T00:00:00Z,2021-11-18T23:59:00Z,
+Campbell Scientific,CR6,3066,CR6.Std.11.01,2021-11-19T00:00:00Z,9999-01-01T00:00:00Z,
+Campbell Scientific,CR6,4108,CR6.Std.11.01,1901-01-01T00:00:00Z,9999-01-01T00:00:00Z,
+Campbell Scientific,CR6,4109,CR6.Std.09.02,2021-10-15T00:00:00Z,9999-01-01T00:00:00Z,
+Campbell Scientific,CR6,4110,CR6.Std.04,2018-08-28T00:00:00Z,2021-11-18T23:59:00Z,
+Campbell Scientific,CR6,4110,CR6.Std.11.01,2021-11-19T00:00:00Z,9999-01-01T00:00:00Z,
+Campbell Scientific,CR6,4111,CR6.Std.11.01,2021-17-11T14:10:00Z,9999-01-01T00:00:00Z,
```

Comment:
Hi @ricardojohansson , this entry has the wrong month (`17`), that is why the pr is failing

Reply:
Thanks @elidana  - I've fixed that... but it's still failing

Reply:
Yes, I can see new and different error messages now popping up, but at least this is one less thing